scss/widgets/base/_checkBox.scss

```diff
@@ -41,13 +43,13 @@
 }
 
 .dx-checkbox-text {
-  display: inline-block;
-  vertical-align: middle;
```

Comment:
is it ok? shoudn't we use `align-items` for flex container or smth like this?

Reply:
not sure, as for me now it looks just like before